frontend/app/helpers/path-helper.js

```diff
@@ -76,6 +76,12 @@ module.exports = function() {
     queryPath: function(queryIdentifier) {
       return '/queries/' + queryIdentifier;
     },
+    repositoryPath: function(projectIdentifier) {
+      return PathHelper.projectPath(projectIdentifier) + '/repository';
+    },
+    revisionPath: function(projectIdentifier, revisionIdentifier) {
+      return PathHelper.repositoryPath(projectIdentifier) + '/revision/' + revisionIdentifier;
+    },
```

Comment:
I already suggested to have this link returned from the revisions endpoint instead:https://github.com/opf/openproject/pull/3326